primefaces/src/main/resources/META-INF/resources/primefaces/datepicker/0-datepicker.js

```diff
@@ -1344,7 +1349,7 @@
 
         renderTitleMonthElement: function (month, index) {
             if (this.options.monthNavigator && this.options.view !== 'month' && index === 0) {
-                return '<select class="ui-datepicker-month">' + this.renderTitleOptions('month', this.options.locale.monthNamesShort, month) + '</select>';
+                return '<select class="ui-datepicker-month" aria-label="'+this.options.locale.select+' '+this.options.locale.month+'">' + this.renderTitleOptions('month', this.options.locale.monthNamesShort, month) + '</select>';
```

Comment:
Why not simply do this?
```suggestion
                return '<select class="ui-datepicker-month" aria-label="'+this.options.locale.month+'">' + this.renderTitleOptions('month', this.options.locale.monthNamesShort, month) + '</select>';
```

Reply:
You think its OK to just read out "Month" and not "Select Month" ?  I would be OK with that?

Reply:
Sure. For email input my label is "Email" and not "Input email". Same thing.